lib/hterm.js

```diff
@@ -155,7 +155,7 @@ hterm.Terminal.prototype.setFontSize = function (px) {
     .unicode-node {
       display: inline-block;
       vertical-align: top;
-      width: ${this.scrollPort_.characterSize.width}px;
+      width: 1em;
```

Comment:
This looks kind of dangerous as unicode characters can be both wide and narrow. Also `1em` is not precise and would therefore not align with the grid.